tests/test_runner/test_parallel.py

```diff
@@ -10,6 +10,17 @@
     tblib = None
 
 
+class ExceptionThatFailsUnpickling(Exception):
+    """
+    This class fails unpickling with an error about incorrect arguments passed
```

Comment:
I would clarify, "After pickling, this class" or "This class fails unpickling (after pickling) with".